src/runtime/tests/runtime-test.ts

```diff
@@ -61,4 +62,44 @@ describe('Runtime', () => {
     const actual = await Runtime.loadManifest('./src/runtime/tests/artifacts/test.manifest', loader, registry);
     assertManifestsEqual(actual, expected);
   });
+  it.only('runs arcs', async () => {
+    const runtime = Runtime.getRuntime();
+    assert.equal(runtime.arcById.size, 0);
+    const arc = runtime.runArc('test-arc', 'volatile://');
+    assert.isNotNull(arc);
+    assert.equal(runtime.arcById.size, 1);
```

Comment:
Optional: I think this is a little more explicit: `assert.hasAllKeys(runtime.arcById, ['test-arc'])`

Reply:
updated